libs/fluid-elements/checkbox/src/lib/checkbox.scss

```diff
@@ -0,0 +1,102 @@
+@import '~@dynatrace/fluid-design-tokens/global/spacing';
+@import '~@dynatrace/fluid-design-tokens/global/typography';
+
+:host {
+  /**
+   * Legibility definitions should probably be
+   * shipped or imported from a core
+  */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-rendering: optimizeLegibility;
+}
+
+/**
+* Disabled state
+*/
+:host([disabled]) {
+  pointer-events: none;
+
+  .fluid-label {
+    color: var(--color-neutral-150);
+    opacity: 0.5;
+  }
+
+  .fluid-svg-checkbox {
+    opacity: 0.5;
+  }
+}
+
+.fluid-checkbox {
+  margin-left: -$fluid-spacing--2x-small;
+  display: flex;
+}
+
+.fluid-label {
+  @include fluid-body();
+  color: var(--color-neutral-150);
+  margin-left: $fluid-spacing--x-small;
+}
+
+label {
+  cursor: pointer;
+  display: inline-flex;
+  align-items: center;
+
+  &:hover .fluid-svg-checkbox-background {
+    opacity: 0.1;
+    transform: scale(1);
+  }
+}
+
+.fluid-checkbox-input {
+  display: none;
```

Comment:
This might not be optimal for screen readers and a11y - This article is quite an interesting read. https://www.sarasoueidan.com/blog/inclusively-hiding-and-styling-checkboxes-and-radio-buttons/

Reply:
Very good point, resolved in the next commit.